cape-ph/cape-cod
# limit static apps to top level domains

# TO TEST

*NOTE: This has been deployed*

* get on the VPN
* go to https://analysis-pipelines.cape-dev.org and poke around the site making sure things work as expected
* manually add any valid path in the s3 bucket to the root URL above and make sure you get the file.
* manually add any invalid path (not in the s3 bucket) to the root URL above and make sure you get an error.
* manually add any valid directory in the s3 bucket and end with a `/` to make sure you are given an error about a missing `index.html` file
